fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -5251,14 +5251,16 @@ class FileBackupAgentImpl {
 			oldRestore.clear(tr);
 		}
 
-		state int index;
-		for (index = 0; index < restoreRanges.size(); index++) {
-			KeyRange restoreIntoRange = KeyRangeRef(restoreRanges[index].begin, restoreRanges[index].end)
-			                                .removePrefix(removePrefix)
-			                                .withPrefix(addPrefix);
-			RangeResult existingRows = wait(tr->getRange(restoreIntoRange, 1));
-			if (existingRows.size() > 0 && !onlyApplyMutationLogs) {
-				throw restore_destination_not_empty();
+		if (!onlyApplyMutationLogs) {
```

Comment:
If we are doing log-only restore, not need to check if target key range is empty. For full restore case, it may call into blob migrator and throw errors.